src/routes/dashboard/containers/DashboardContainer.js

```diff
@@ -133,11 +134,16 @@ const DashboardContainer = (props: ContainerPropsType) => {
     navigateToExtras={createNavigateToExtras(dispatch, rest.navigation)} />
 }
 
-export default withRouteCleaner<PropsType>(
-  connect<PropsType, OwnPropsType, _, _, _, _>(mapStateToProps, mapDispatchToProps)(
-    omitNavigation<PropsType>(
-      withPayloadProvider<ContainerPropsType, RefreshPropsType>(refresh)(
-        DashboardContainer
-      )
-    )
-  ))
+type RestType = $Diff<PropsType, OwnPropsType>
```

Comment:
Did this fix some flow error?

Reply:
This is used to remove `t` and `navigation` from props (passed through `ownProps`). Before just `navigation` was removed via `omitNavigation`